Gems/Atom/Asset/ImageProcessingAtom/Assets/Config/ImageBuilder.settings

```diff
@@ -201,6 +201,11 @@
             "_specular": [ "Reflectance" ],
 			"_SpecularReflectance": [ "Reflectance" ],
 			"_thickness": [ "Reflectance" ],
+			"_linear": [ "Reflectance" ],
+			"_sm": [ "Reflectance" ],
+			"_splat": [ "Reflectance" ],
+			"_splatmap": [ "Reflectance" ],
```

Comment:
Should these go under "Gradient" instead of "Reflectance"?

Reply:
moved them